YMCache.podspec

```diff
@@ -6,14 +6,11 @@ Pod::Spec.new do |s|
   s.license          = 'MIT'
   s.author           = { 'adamkaplan' => 'adamkaplan@yahoo-inc.com' }
   s.source           = { :git => 'https://github.com/yahoo/YMCache.git', :tag => s.version.to_s }
-  s.social_media_url = 'https://twitter.com/adkap'
 
-  s.ios.deployment_target = '8.0'
-  s.osx.deployment_target = '10.10'
-  s.tvos.deployment_target = '11.0'
-  s.watchos.deployment_target = '3.0'
-
-  s.requires_arc = true
+  s.ios.deployment_target = '13.0'
```

Comment:
Can this be updated in the [Package.swift](https://github.com/yahoo/YMCache/blob/master/Package.swift#L8) as well so they're in sync?